packages/server/validators/events.js

```diff
@@ -164,16 +170,20 @@ const updateEventValidator = [
     .toDate()
     .withMessage("End date is not in a valid date format")
     .custom(async (endDate, { req }) => {
-      if (req.body.startDate) {
-        if (endDate < req.body.startDate) {
-          throw new Error("End date is before new start date");
-        }
-      } else {
+      //Check for when new end date is given but no start date
+      if (!req.body.startDate) {
         const currEvent = await EventController.getEvent(req.params.eventId);
-        if (currEvent.startDate != null && endDate < currEvent.startDate) {
-          throw new Error("End date is before start date");
+
+        //Check if current start date is null
+        if (currEvent.startDate == null) {
+          throw new Error("End date updated but start date is null");
+        }
```

Comment:
Same thing as my earlier message about the update start date error message. This error message implies we updated the event's end date when we didn't, the request failed. Change this error message to something more like "Cannot update end date if current start date is null".